src/Navigation.tsx

```diff
@@ -74,6 +76,14 @@ function MainStack({navigation}: Props) {
 
   const headerHeightIncludingSafeArea = useHeaderHeight()
 
+  const {loginState} = useContext(AuthContext)
+
+  useEffect(() => {
+    if (loginState !== LoginState.LoggedOut) {
+      navigation.navigate(SCREENS.HOME)
+    }
+  }, [loginState])
```

Comment:
I don't think this handles this as well as it should. I think we should keep the logo screen up until it has loaded, and then navigate to the appropriate page. Otherwise when logged in we can briefly see the splash screen when we are logged in.

Which we could handle with:

```initialRouteName={loggedIn ? SCREENS.HOME : SCREENS.SPLASH}```